source/_integrations/light.markdown

```diff
@@ -27,6 +27,8 @@ The field transition is optional and can be omitted.
 
 The `.default` suffix should be added to the entity identifier of each light to define a default value, e.g., for `light.ceiling_2` the `profile` field is `light.ceiling_2.default`. To define a default for all lights, the identifier `group.all_lights.default` can be used. Individual settings always supersede the `all_lights` default setting.
 
+the `color_x` and `color_y` can be found in the`xy_color` entity attribute. 
```

Comment:
There is spacing inconsistency

Reply:
This sentence feels misplaced? It feels out of context here, wdyt?